app/scripts/controllers/swaps.js

```diff
@@ -256,6 +260,7 @@ export default class SwapsController {
     isPolledRequest,
   ) {
     const { chainId } = fetchParamsMetaData;
+    const networkClientId = this._getNetworkClientId();
```

Comment:
Apologies, I think I missed this new method in the core PR.

Ideally `networkClientId` is an optional argument, meaning the clients that aren't fully multi-chain can instead just pass the `chainId` and we use just that to find a suitable network client in the background.

Meaning this additional constructor option wouldn't be needed.

Reply:
There was a small error that needed fixing in the `TransactionController`, so I've updated the signature to make these optional at the same time.